src/js/utils/JobUtil.js

```diff
@@ -123,7 +158,12 @@ const JobUtil = {
   },
 
   createFormModelFromSchema(schema, job = new Job()) {
-    return getMatchingProperties(job, schema.properties);
+    let newSchemaProperties = {};
+    newSchemaProperties = schema.properties;
+    console.log(newSchemaProperties);
+    addTrans(newSchemaProperties);
```

Comment:
How come we have to add translation elements here instead of just having jsx elements (`Trans`) for all the properties that end up getting manually translated in the schema files? It seems some of the properties are already jsx elements (some of the `description`s and a `helpBlock`)

Reply:
You mean instead of having for example `description: i18nMark("The job ID")` in `General.js`, we should have `description: <Trans render="span">The job ID</Trans>`? This is how I started doing it at first, but then if we decide to change the text to "Job Identification" for example we have to add a `i18nMark` to add the new text to the message catalog. I considered having both `i18nMark` and a `<Trans>`, but I couldn't find any examples of doing it like that in the code, so I dropped this idea.